src/models/room-state.ts

```diff
@@ -976,19 +976,21 @@ export class RoomState extends TypedEventEmitter<EmittedEvents, EventHandlerMap>
      * Note: if an m.predecessor event is used, eventId is null since those
```

Comment:
Comment should be updated to say undefined instead of null, but also I just realised this comment is out of date, because MSC3946 is being updated to allow eventId, and this is what the code supports, so it should read:

```
Note: if an m.predecessor event is used, eventId may be undefined since last_known_event_id is optional.
```